src/status_im/keycard/core.cljs

```diff
@@ -589,7 +588,8 @@
               (when (and (= card-state :profile/profile)
                          (= flow :import))
                 (if (common/find-multiaccount-by-key-uid db key-uid)
-                  (multiaccounts.recover/show-existing-multiaccount-alert key-uid)
+                  ;; reimplement
```

Comment:
please consider creating an issue and TODO etc 👍

Reply:
no need in issues and TODO, keycard will be implemented